src/WorkItemMigrator/JiraExport/JiraProvider.cs

```diff
@@ -28,34 +28,55 @@ public enum DownloadOptions
 
         private readonly string JiraApiV2 = "rest/api/2";
 
+        private Dictionary<string, string> JiraNameFieldCache = null;
+
+        private Dictionary<string, string> JiraKeyFieldCache = null;
+
         readonly Dictionary<string, string> _userEmailCache = new Dictionary<string, string>();
 
-        public Jira Jira { get; private set; }
+        private readonly IJiraRestClient _restClient;
```

Comment:
Where are these Service interfaces defined? I can't find the definitions

        private readonly IJiraRestClient _restClient;
        private readonly IIssueLinkService _issueLinkService;
        private readonly IIssueFieldService _issueFieldService;
        private readonly IIssueService _issueService;
        private readonly IJiraUserService _jiraUserService;

Reply:
They are coming from the Atlassian.Jira dependency and namespace:

![image](https://user-images.githubusercontent.com/1309462/189715435-050d5ed3-f55b-479c-86b0-99661c42b675.png)

Reply:
Ah, of course. I'm stupid :)